PSReadLine/KeyBindings.cs

```diff
@@ -229,24 +229,28 @@ void SetDefaultWindowsBindings()
                 { Keys.ShiftF3,                MakeKeyHandler(CharacterSearchBackward,   "CharacterSearchBackward") },
                 { Keys.F8,                     MakeKeyHandler(HistorySearchBackward,     "HistorySearchBackward") },
                 { Keys.ShiftF8,                MakeKeyHandler(HistorySearchForward,      "HistorySearchForward") },
-                { Keys.PageUp,                 MakeKeyHandler(ScrollDisplayUp,           "ScrollDisplayUp") },
-                { Keys.PageDown,               MakeKeyHandler(ScrollDisplayDown,         "ScrollDisplayDown") },
-                { Keys.CtrlPageUp,             MakeKeyHandler(ScrollDisplayUpLine,       "ScrollDisplayUpLine") },
-                { Keys.CtrlPageDown,           MakeKeyHandler(ScrollDisplayDownLine,     "ScrollDisplayDownLine") },
                 // Added for xtermjs-based terminals that send different key combinations.
                 { Keys.AltD,                   MakeKeyHandler(KillWord,                  "KillWord") },
                 { Keys.CtrlAt,                 MakeKeyHandler(MenuComplete,              "MenuComplete") },
                 { Keys.CtrlW,                  MakeKeyHandler(BackwardKillWord,          "BackwardKillWord")},
             };
 
             // Some bindings are not available on certain platforms
+            // For example, PageUp/PageDown and CtrlPageUp/CtrlPageDown bindings are supported on Windows only because
```

Comment:
Rather than `for example` above (which I feel invites further entries to skip an explanation of why they are platform-specific), maybe it's preferable to put explanatory comments above entries (or whitespace-separated groups of entries)

Reply:
Agreed. Will put this comment above the entries.